installer/EnhancedDiscordUI/Form1.cs

```diff
@@ -260,11 +269,27 @@ private async void continueInstall(Process proc)
                     basePath = Path.Combine(basePath, ".config");
                 }
             }
-            string[] pathPieces = { basePath, release, appVersion, "modules", "discord_desktop_core", "index.js" };
-            // the base "appdata" folder ^        ^       ^
-            // i.e. "discord" or "discordcanary" -'       |
-            // i.e. "0.0.300" or "0.0.204" ---------------'
+            string[] pathPieces;
+            if (is1x)
+            {
+                // 1.x
+                pathPieces = new string[] { basePath, release, "app-" + appVersion, "modules", "discord_desktop_core-1", "discord_desktop_core", "index.js" };
```

Comment:
This is where I suggest you change to reflect Discord's module versions, so it doesn't break on most of them.  As they update modules more often than the host.